JFrogVSExtension/Utils/Util.cs

```diff
@@ -185,6 +193,32 @@ public static Component ParseDependencies(Dependency dep, Dictionary<string, Art
             return comp;
         }
 
+        // Make sure that all of the component's dependencies are added to the DataService.
+        // This scenario might happen as a package may appear in the dependencies-tree numerous times with different dependencies.
+        // This is due to a possible situation of multiple subprojects under the same solution,
+        // where the same package appears in several projects but its dependencies varies between the packages.config files of the subprojects.
```

Comment:
// where the same package appears in several projects but its dependencies varies between the packages.config files of the subprojects.
-->
// where the same package appears in several projects, but for some projects, some dependencies are missing in the packages.config.

Reply:
Changed.